conceptnet5/vectors/evaluation/bias.py

```diff
@@ -196,15 +195,16 @@ def measure_bias(frame):
     - Coarse-grained ethnicity
     - Religious beliefs
     """
-    vsw = VectorSpaceWrapper(frame=frame)
-    vsw.load()
+    # Assume English for DataFrames that don't have language tags.
+    if '/c/' not in frame.index[0]:
+        frame.index = ['/c/en/%s' % label for label in frame.index]
```

Comment:
I think his shouldn't be necessary, as `get_vector()` have an ability to look up words in indices made up of concepts or not.

Although, as I was looking to confirm that, I noticed that `get_vector` checks what sort of index it deals with by looking if `frame.index[0].startswith('/')`, which we should probably change to `frame.index[0].startswith('/c/')`.